pkg/tool/kube/client/cluster.go

```diff
@@ -48,6 +50,21 @@ func Cluster() cluster.Cluster {
 	return c
 }
 
+func NewClientSet() *kubernetes.Clientset {
+	if k != nil {
+		return k
+	}
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		panic(err)
```

Comment:
It is better to `return error` than `panic` to let the user decide how to handle the exception.